motor/core.py

```diff
@@ -1080,6 +1083,7 @@ def get_io_loop(self):
 
 class AgnosticBaseCursor(AgnosticBase):
     """Base class for AgnosticCursor and AgnosticCommandCursor"""
+    _async_close  = AsyncRead(attr_name='close')
```

Comment:
The cursor close changes make motorcursor.close() wait for kill cursors rather than deferring the killCursors to the background thread.